src/main/fc/fc_msp.c

```diff
@@ -822,6 +823,20 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
         }
         break;
 
+#ifdef VTX_SMARTAUDIO
```

Comment:
RC6 is not the time to be doing something like this. There may be a case to do a 3.1.1 release to get the VTX changes in.